app/models/application_letter.rb

```diff
@@ -83,4 +83,15 @@ def status_cannot_be_changed
       errors.add(:event, "Die Bewerbungen wurden bereits bearbeitet, eine Statusänderung ist nicht mehr erlaubt.")
     end
   end
+
+  # Returns the age of the user based on the date the event starts
+  #
+  # @param none
+  # @return [Int] for age as number of years
+  def applicant_age_when_event_starts
+    start_date = event.start_date
+    birth_date = user.profile.birth_date
+    start_date_is_before_birthday = start_date.month > birth_date.month || (start_date.month == birth_date.month && start_date.day >= birth_date.day)
+    return start_date.year - birth_date.year - (start_date_is_before_birthday ? 0 : 1)
```

Comment:
This method is very similar to [User#older_than_required_age_at_start_date_of_event?](https://github.com/hpi-swt2/workshop-portal/blob/dev/app/models/user.rb#L80) and [Profile#age](https://github.com/hpi-swt2/workshop-portal/blob/dev/app/models/profile.rb#L33). I think it should be a good idea to provide this functionality in a module or something like this?